### PR TITLE
 VictorGrig USBscope PID pull request #3

### PR DESCRIPTION
**Pull request by VictorGrig**
General clean-up. My project has been moved into its own new [repository](https://github.com/VictorGrig/USBscope) as you adviced at #77, it contains:
- C source code for AVR MCU;
- C# source code for desktop application;
- AD 14.3 PCB-project;
- SL 6 PCB-project;
- Current executable file.

### CODE REVIEW

**Comment by Arachnid** [comment]
Looks good, thanks.

Your repository has two LICENSE files, one CC0 and the other MIT. Were you intending to dual-license under both?

**Comment by VictorGrig** [comment]
Ah, sorry for my inattention. CC0 license has been copied by mistake, The MIT License file is primary now.

File: 1209/EBA7/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: USBscope
+owner: Victor Grigoryev
```
Comment (Arachnid) [inline 1209/EBA7/index.md]: The 'owner' field needs to match the directory name of the directory in which your org file is ('USBscope').

Comment (VictorGrig) [inline 1209/EBA7/index.md]: Finally [index.md](https://github.com/VictorGrig/pidcodes.github.com/blob/VictorGrig-USBscope-PID-pull-request/1209/EBA7/index.md) should be like this, isn`t it?

```markdown
---
layout: pid
title: USBscope
owner: USBscope
license: MIT
site: http://yveslebrac.blogspot.com/2008/10/cheapest-dual-trace-scope-in-galaxy.html
source: https://github.com/VictorGrig/USBscope
---
```

**Comment by Arachnid** [comment]
As it stands, the device will be identified in the PID list as "USBScope USBScope". If you're happy with that, I'm happy to merge this, but if you'd prefer something more descriptive, you should change the title of the org to something more descriptive, such as your name. It would probably make sense to change the directory it's in if you do that too.

**Comment by VictorGrig** [comment]
Okay, I got confused little bit =) 

Let's set my identifier as `Victor Grigoryev USBscope`, where  `USBscope` - project title and `Victor Grigoryev` - my name :D What should I change to get such an ID?

**Comment by Arachnid** [comment]
Set the "title" field in the org file to "Victor Grigoryev". For the sake of clarity and any future pids, rename the directory it's in to something like "victor-grigoryev", and update the "owner" field in the PID file to match.

**Comment by VictorGrig** [comment]
Got it.
- org-folder has been renamed to `VictorGrigoryev`
- org-file and PID-file title has been changed to `VictorGrigoryev`

**Pull request merged**